<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
    color: rgba(255, 255, 255, 10) ;
	background-color: qlineargradient(spread:pad, x0:0, y0:2, x2:0, y2:1.1, stop:0 rgba(255, 121, 14, 255), stop:1 rgba(80, 35, 46, 255));
	
	opacity:80;
	position: relative;
    background-position: center;
    background-repeat: no-repeat;
    
}

QPushButton {
 background-color: qlineargradient(spread:pad, x0:0, y0:2, x2:0, y2:1.1, stop:0 rgba(80, 35, 46, 255), stop:1 rgba(255, 121, 14, 255));
 color: black;
font-family:Epitet-SemiBold;
font-size: 16px;
margin:2px;
 border-radius:10px;
}
QPushButton:Hover {
 background-color: rgb(255, 121, 14);
 color: white;
font-family:Epitet-SemiBold;
font-size: 16px;
margin:2px;
 border-radius:10px;
}
QPushButton:Pressed {
 background-color: rgb(80, 35, 46);
 color: black;
font-family:Epitet-SemiBold;
font-size: 16px;
margin:2px;
 border-radius:10px;
}</string>
  </property>
  <widget class="QPushButton" name="Buscar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Epitet-SemiBold</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Buscar
Hardware</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NuevoHardware">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Epitet-SemiBold</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nuevo
Hardware</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModificarHardware">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Epitet-SemiBold</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Modificar
Hardware</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EliminarHardware">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Epitet-SemiBold</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eliminar
Hardware</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BuscarID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly|Qt::ImhUrlCharactersOnly</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>821</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>Listado de Hardware</string>
   </property>
   <widget class="QTableWidget" name="TablaHardware">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>801</width>
      <height>371</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>ID Hardware</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo de Hardware</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Serial</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="DetallesHardware">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Epitet-SemiBold</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Detalles 
Hardware</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
